build

```diff
@@ -760,66 +760,35 @@ mkdir -p "$DESTINATION/tmp"
 echo "TARGET=${UFCONTEXT_HEADER_NAME}" > $DESTINATION/tmp/uFcontext.mk
 
 # Create the bin, lib, include, etc. links:
-WINNAME=`echo $VERSION | awk -F- '{print $2}'`
-if [ "x_$WINNAME" = "x_win64" -o "x_$WINNAME" = "x_win" ]
-then
-#Win64 version needs special compilers and *copied* (not linked)
-# source files.
-	cp $src/win/system_ln $DESTINATION/tmp
-	cp $src/win/unistd.h $DESTINATION/tmp
-	echo "Compiling createlink.cpp ..."
-	case "$COMPILER" in
-		gcc*|clang*)
-			createlinkcc="$COMPILER"
-			;;
-		*)
-			createlinkcc="./unix2nt_cc"
-			;;
-	esac
-	(cd $src/win; $createlinkcc -c createlink.cpp -o createlink.o -D_WIN32_WINNT=0x0500; $createlinkcc createlink.o -o createlink.exe)
-
-	if test ! -x $src/win/createlink.exe
-	then
-		echo "VC++ is not properly installed!"
-		exit 1
-	fi
-	chmod +x $DESTINATION/tmp/system_ln
-	cp $src/win/gathertree.local   $DESTINATION/tmp
-	cp $src/win/gatherflat.local   $DESTINATION/tmp
-else
-	cat > $DESTINATION/tmp/system_ln <<EOF
-#!/bin/sh
-ln -f -s \$@
-EOF
-  chmod +x $DESTINATION/tmp/system_ln
-  newdirlist='bin lib'
-  [ -n "$BUILD_SHARED" ] && newdirlist="$newdirlist lib_so"
-  newdirlist="$newdirlist include tmp"
-  for newdir in $newdirlist
-  do
-    Echo "Soft-linking over $newdir"
-    if [ -r $newdir ]
-    then
-      rm -fr $newdir || exit 1
```

Comment:
Should this return check still be there?

Reply:
Not sure. I guess `rm -f` can not fail, so the `exit 1` is never executed.